templates/download/server/download.html

```diff
@@ -73,7 +73,7 @@ <h2 class="p-heading--three">Get Ubuntu Server pro tips</h2>
     <div class="row u-equal-height">
       <div class="col-6">
         <h3>Enterprise Support for Ubuntu Server</h3>
-        <p>Looking for commercial support for Ubuntu Server? Ubuntu Advantage for Infrastructure (UA-I) provides production-grade SLAs, phone and ticket support, Extended Security Maintenance (ESM) and more.</p>
+        <p>Looking for commercial support for Ubuntu Server? Ubuntu Advantage for Infrastructure (UA-I) provides production-grade SLAs, phone and ticket support, Extended Security Maintenance for Infrastructure (ESM Infra) and more.</p>
```

Comment:
I missed this one in my drive-by:
```suggestion
        <p>Looking for commercial support for Ubuntu Server? Ubuntu Advantage for Infrastructure (UA Infra) provides production-grade SLAs, phone and ticket support, Extended Security Maintenance for Infrastructure (ESM Infra) and more.</p>
```